internal/proc/user32.go

```diff
@@ -1,165 +1,167 @@
-package proc

```

Comment:
Hi,

Thank you for your contribution.

Apparently you changed all the linebreaks of the files, which are standard Windows (CR+LF) to standard Unix (LF), so the diffs are huge. Probably your editor is configured to force Unix (LF).

Do you mind to verify this?